zcutil/build.sh

```diff
@@ -73,9 +76,17 @@ then
     shift
 fi
 
+# If --disable-rust is the next argument, disable Rust code:
```

Comment:
Can you push to this PR with what you're talking about? I'm on my laptop. :blush: